### PR TITLE
Strip out script tags from embedded js source

### PR DESCRIPTION
**Pull request by davidtron**
Work around for #2 this replaces <script> and <\script> from the js that is embedded in the html files

### CODE REVIEW

File: src/lib/index.ts [hunk 1]
```diff
@@ -45,8 +45,12 @@ class AssetEntry {
         }
     }
     private applyScript(tag: HTMLScriptElement) {
+        const start = /<script>/gi;
+        const end = /<\/script>gi;
```
Comment (otiai10) [inline src/lib/index.ts:49]: missing `/` at the tail?

File: src/lib/index.ts [hunk 2]
```diff
@@ -45,8 +45,12 @@ class AssetEntry {
...
         tag.removeAttribute('src');
-        tag.innerHTML = this.content.toString();
+        var js = this.content.toString();
```
Comment (otiai10) [inline src/lib/index.ts:51]: Do we use `js` variable?

File: src/lib/index.ts [hunk 3]
```diff
@@ -45,8 +45,12 @@ class AssetEntry {
...
+        js = content.replace(start,' ').replace(end,' ');
```
Comment (otiai10) [inline src/lib/index.ts]: Does it have to be `' '` than `''`?

**Review by otiai10 — COMMENTED** [review]
Thank you very much!

Because just removing `<script>` and `</script>` can break the original purpose of the JavaScript, I still want to see the reproducible HTML file and see what is actually happening.

Is it possible to give me some HTML as examples?

File: src/lib/index.ts (hunk 2 above)
Comment (davidtron) [reply src/lib/index.ts:51]: sorry - thats a mistake, i had to retype the fix from my work computer to my personal one, updating the fix now

File: src/lib/index.ts (hunk 3 above)
Comment (davidtron) [reply src/lib/index.ts]: Not really - my understanding was this is in the content of the js that gets embedded - i don't think it matters.

File: src/lib/index.ts (hunk 1 above)
Comment (davidtron) [reply src/lib/index.ts:49]: yep - was retyping the fix from my work computer and bungled it.... fixed in latest push

**Comment by davidtron** [comment]
[DocumentCategoryJson.txt](https://github.com/otiai10/jacoco-html-asset-bundle/files/6700260/DocumentCategoryJson.txt)
I've updated a redacted sample of a java.html file that contains the issue.  In this case it's sufficient to remove the </script> on line 287 to make the file render correctly.

**Comment by davidtron** [comment]
To be honest, this PR was just a quick and dirty fix (I'm not really a ts dev) a nicer solution would be to remove all the comments from the javascript libraries before inlining (I don't know how feasible it would be to inline the minified js which I assume has comments stripped out)

File: src/lib/index.ts (hunk 3 above)
Comment (davidtron) [reply src/lib/index.ts]: Pushed '' in last fix

**Comment by otiai10** [comment]
Thanks a lot! It helps.
Now I merge this PR but keep looking for a nicer way, and please let me involve you into that activity ;)

**Comment by davidtron** [comment]
Happy to look at alternate solutions, also, please let me know once this is merged and a new version is pushed to npm

**Comment by otiai10** [comment]
@davidtron Please check #7

**Pull request merged**